### PR TITLE
Decrease request dexToken count for e2e tests

### PR DESCRIPTION
**Pull request by StanislavStefanov**
**Description**

Changes proposed in this pull request:
- request dexToken and create dexGraphQLClient only once in main_test for director, gateway and external-services-mock components
- fix deploy-tests-on-minikube target
- fix run target for every component

### CODE REVIEW

File: tests/director/tests/main_test.go [hunk 1]
```diff
@@ -14,9 +20,13 @@ import (
...
 )
 
-var conf = &config.DirectorConfig{}
+var (
+	conf             = &config.DirectorConfig{}
+	dexGraphQLClient *graphql.Client
+)
 
 func TestMain(m *testing.M) {
+	fmt.Println("yoooooooooooo")
```
Comment (desislavaa) [inline tests/director/tests/main_test.go:29]: yoo

File: tests/external-services-mock/tests/async_api_delete_test.go [hunk 2]
```diff
@@ -78,9 +69,6 @@ func TestAsyncAPIDeleteApplicationWithAppTemplateWebhook(t *testing.T) {
 	}
 
 	t.Log("Get Dex token")
```
Comment (desislavaa) [inline tests/external-services-mock/tests/async_api_delete_test.go:71]: delete log as well

File: tests/external-services-mock/tests/async_api_delete_test.go [hunk 3]
```diff
@@ -126,9 +114,6 @@ func TestAsyncAPIDeleteApplicationPrioritizationWithBothAppTemplateAndAppWebhook
...
```
Comment (desislavaa) [inline tests/external-services-mock/tests/async_api_delete_test.go:116]: delete log

File: tests/director/tests/main_test.go [hunk 4]
```diff
@@ -27,11 +35,18 @@ func TestMain(m *testing.M) {
 
 	testctx.Tc, err = testctx.NewTestContext()
 	if err != nil {
-		log.Fatal(err)
+		log.Fatal(errors.Wrap(err, "while getting dex token"))
```
Comment (ataleksandrov) [inline tests/director/tests/main_test.go:38]: This error message should be on the next `log.Fatal`

File: tests/director/tests/main_test.go [hunk 5]
```diff
@@ -27,11 +35,18 @@ func TestMain(m *testing.M) {
 
 	testctx.Tc, err = testctx.NewTestContext()
```
Comment (ataleksandrov) [inline tests/director/tests/main_test.go:36]: As far as I remember we discussed, in the PR with the new structure of the e2e tests, that this assignment is unnecessary as test.Tc is initialized in the init method of the package. Could we remove it?

**Review by dzahariev — APPROVED** [review]

**Pull request merged**